Unable to create application

```
FATAL EXCEPTION: main
Process: io.treehouses.remote, PID: 4755
java.lang.RuntimeException: Unable to create application io.treehouses.remote.MainApplication: java.lang.IllegalArgumentException: Cannot register a type that does not implement the default constructor!
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6746)
	at android.app.ActivityThread.access$1400(ActivityThread.java:237)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1924)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7741)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:966)
Caused by: java.lang.IllegalArgumentException: Cannot register a type that does not implement the default constructor!
	at com.parse.ParseObjectSubclassingController.registerSubclass(SourceFile:181)
	at com.parse.ParseObject.registerSubclass(SourceFile:5)
	at com.parse.ParseObject.registerParseSubclasses(SourceFile:3)
	at com.parse.Parse.initialize(SourceFile:8)
	at com.parse.Parse.initialize(SourceFile:1)
	at io.treehouses.remote.MainApplication.onCreate(SourceFile:62)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1206)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6741)
	at android.app.ActivityThread.access$1400(ActivityThread.java:237) 
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1924) 
	at android.os.Handler.dispatchMessage(Handler.java:106) 
	at android.os.Looper.loop(Looper.java:223) 
	at android.app.ActivityThread.main(ActivityThread.java:7741) 
	at java.lang.reflect.Method.invoke(Native Method) 
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:966) 
```

> ```
> FATAL EXCEPTION: main
> Process: io.treehouses.remote, PID: 4755
> java.lang.RuntimeException: Unable to create application io.treehouses.remote.MainApplication: java.lang.IllegalArgumentException: Cannot register a type that does not implement the default constructor!
> 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6746)
> 	at android.app.ActivityThread.access$1400(ActivityThread.java:237)
> 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1924)
> 	at android.os.Handler.dispatchMessage(Handler.java:106)
> 	at android.os.Looper.loop(Looper.java:223)
> 	at android.app.ActivityThread.main(ActivityThread.java:7741)
> 	at java.lang.reflect.Method.invoke(Native Method)
> 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
> 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:966)
> Caused by: java.lang.IllegalArgumentException: Cannot register a type that does not implement the default constructor!
> 	at com.parse.ParseObjectSubclassingController.registerSubclass(SourceFile:181)
> 	at com.parse.ParseObject.registerSubclass(SourceFile:5)
> 	at com.parse.ParseObject.registerParseSubclasses(SourceFile:3)
> 	at com.parse.Parse.initialize(SourceFile:8)
> 	at com.parse.Parse.initialize(SourceFile:1)
> 	at io.treehouses.remote.MainApplication.onCreate(SourceFile:62)
> 	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1206)
> 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6741)
> 	at android.app.ActivityThread.access$1400(ActivityThread.java:237) 
> 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1924) 
> 	at android.os.Handler.dispatchMessage(Handler.java:106) 
> 	at android.os.Looper.loop(Looper.java:223) 
> 	at android.app.ActivityThread.main(ActivityThread.java:7741) 
> 	at java.lang.reflect.Method.invoke(Native Method) 
> 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
> 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:966) 
> ```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -27,16 +27,16 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled true
-            shrinkResources true
+            minifyEnabled false
+            shrinkResources false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
         //Uncomment for debugging
-//        debug {
-//            minifyEnabled true
-//            shrinkResources true
-//            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-//        }
+        debug {
+            minifyEnabled false
+            shrinkResources false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
 
     }
     lint {

--- a/app/src/main/kotlin/io/treehouses/remote/fragments/dialogfragments/RPIDialogFragment.kt
+++ b/app/src/main/kotlin/io/treehouses/remote/fragments/dialogfragments/RPIDialogFragment.kt
@@ -236,12 +236,12 @@ class RPIDialogFragment : BaseDialogFragment(), DeviceDeleteListener {
         }
 
         fun checkPiAddress(deviceHardwareAddress: String): Boolean {
-            val piAddress: Set<String> = HashSet(listOf("B8:27:EB", "DC:A6:32", "E4:5F:01", "28:CD:C1", "E4:F5:01",
-                                                        "B8-27-EB", "DC-A6-32", "E4-5F-01", "28-CD-C1", "E4-F5-01",
-                                                        "B827.EB", "DCA6.32", "E45F.01", "28CD.C1", "E4F5.01",
-                                                        "b8:27:eb", "dc:a6:32", "e4:5f:01", "28:cd:c1", "e4:f5:01",
-                                                        "b8-27-eb", "dc-a6-32", "e4-5f-01", "28-cd-c1", "e4-f5-01",
-                                                        "b827.eb", "dca6.32", "e45f.01", "28cd.c1", "e4f5.01"))
+            val piAddress: Set<String> = HashSet(listOf("B8:27:EB", "DC:A6:32", "E4:5F:01", "28:CD:C1", "D8:3A:DD",
+                                                        "B8-27-EB", "DC-A6-32", "E4-5F-01", "28-CD-C1", "D8-3A-DD",
+                                                        "B827.EB", "DCA6.32", "E45F.01", "28CD.C1", "D83A.DD",
+                                                        "b8:27:eb", "dc:a6:32", "e4:5f:01", "28:cd:c1", "d8:3a:dd",
+                                                        "b8-27-eb", "dc-a6-32", "e4-5f-01", "28-cd-c1", "d8-3a-dd",
+                                                        "b827.eb", "dca6.32", "e45f.01", "28cd.c1", "d83a.dd"))
             for (item in piAddress) {
                 if (deviceHardwareAddress.contains(item)) return true
             }

